src/components/AddressOverview.vue

```diff
@@ -34,11 +34,11 @@ const systemToken = useChainStore().currentChain.settings.getSystemToken();
         <q-skeleton type="text" class="c-address-overview__skeleton" />
     </q-card-section>
     <q-card-section v-else>
-        <div class="c-address-overview__label"> TLOS {{ $t('pages.balance') }} </div>
+        <div class="c-address-overview__label"> {{ systemToken.symbol }} {{ $t('pages.balance') }} </div>
         <div class="c-address-overview__balance">
             <img
```

Comment:
<img width="205" alt="image" src="https://github.com/user-attachments/assets/8114da72-6196-4207-9491-3765bcfaf6c7">
Can we fix logo placement (align it to the balance value).

Reply:
I see it normally

![image](https://github.com/user-attachments/assets/de81b5fc-7ad1-4a0e-9a35-333e3fa0d308)

What browser are you using?

Reply:
I use chrome

Reply:
can't reproduce it, weird.